go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/longhorn/longhorn-engine
 
 go 1.13
 
+replace github.com/longhorn/backupstore => github.com/PhanLe1010/backupstore v0.0.0-20200704071512-09776745c1ed
```

Comment:
In reference to our discussion, instead of pointing to your fork, which requires a pushed commit.
You can also point to your local dev version on the filesystem, this generates less changes.
And you still will need the `rebase -i` once the backupstore commits are merged into master.

Reply:
Thank you Joshua for the instruction! My current setup is a bit awkward (code on Mac and sync to Linode for testing). I don't do git operation on the Linode machine. So, I guess the only option is pushing backupstore commits to the fork and pointing to it. Is it correct?

In the future, I will transition to use a Linux Virtualbox machine on my Mac to do both coding and testing. At this time, I will follow the guideline you told me. Thank you again.

Reply:
That explains the setup, no worries just figured I point out an alternative that reduces the friction a little bit :)

Reply:
Thank you :D